.github/workflows/ci-artifacts.yml

```diff
@@ -0,0 +1,236 @@
+name: ci-artifacts
+
+on:
+  push:
+    branches:
+      - add-ci-artifacts
+  pull_request:
+
+# For the continuous `ci-artifacts` release
+permissions:
+  contents: write
+
+env:
+  LC_CTYPE: C.UTF-8
+
+jobs:
+  minimal-sdk-artifact:
+    # if: github.event.repository.fork == false
+    runs-on: dennis-windows-arm64
+    steps:
+      - name: TO-DROP Install build dependencies
+        shell: powershell
+        run: |
+          # Download the script to a local path
+          Invoke-WebRequest -Uri "https://raw.githubusercontent.com/dennisameling-org/git-for-windows-automation/github-hosted-runners/azure-self-hosted-runners/post-deployment-script.ps1" -OutFile "$env:TEMP\post-deployment-script.ps1"
+
+          # Set PowerShell execution policy to unrestricted for the current session
+          Set-ExecutionPolicy -Scope Process -ExecutionPolicy Unrestricted
+
+          # Run the script
+          & "$env:TEMP\post-deployment-script.ps1"
+
+          Add-Content $env:GITHUB_PATH "C:\Program Files\Git\bin"
+          Add-Content $env:GITHUB_PATH "C:\Program Files\Git\cmd"
+          Add-Content $env:GITHUB_PATH "C:\Program Files\PowerShell\7"
+      - name: clone git-sdk-arm64
+        shell: bash
+        env:
+          PATH: C:\Program Files\Git\cmd;C:\Program Files\PowerShell\7;${{ env.PATH }}
+        run: |
+          git init --bare git-sdk-arm64.git &&
+          git --git-dir=git-sdk-arm64.git remote add origin https://github.com/${{github.repository}} &&
+          git --git-dir=git-sdk-arm64.git config remote.origin.promisor true &&
+          git --git-dir=git-sdk-arm64.git config remote.origin.partialCloneFilter blob:none &&
+          git --git-dir=git-sdk-arm64.git fetch --depth=1 origin ${{github.sha}} &&
+          git --git-dir=git-sdk-arm64.git update-ref --no-deref HEAD ${{github.sha}}
+      - name: clone build-extra
+        run: git clone --depth=1 --single-branch -b main https://github.com/git-for-windows/build-extra
+      - name: build git-sdk-arm64-minimal-sdk
+        shell: bash
+        run: |
+          sh -x ./build-extra/please.sh create-sdk-artifact --sdk=git-sdk-arm64.git minimal-sdk &&
+          cygpath -aw minimal-sdk/usr/bin >>$GITHUB_PATH
+      - name: compress artifact
+        shell: bash
+        run: (cd minimal-sdk && tar cvf - * .[0-9A-Za-z]*) | gzip -1 >git-sdk-aarch64-minimal.tar.gz
+      - name: upload minimal-sdk artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: minimal-sdk
+          path: git-sdk-aarch64-minimal.tar.gz
+      - name: create zip and 7z SFX variants of the minimal SDK
+        # if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+        if: github.event_name == 'push' && github.ref == 'refs/heads/add-ci-artifacts'
+        shell: bash
+        run: |
+          for path in clangarm64/bin/7z.exe clangarm64/bin/7z.dll clangarm64/lib/7zip/7zCon.sfx clangarm64/bin/libc++.dll

```

Comment:
I had to add `clangarm64/bin/libc++.dll` here, because [I got this error](https://github.com/dennisameling-org/git-sdk-arm64/actions/runs/10862732128/job/30146090626#step:8:24):

```
Error: Process completed with exit code 127.
```

Reply:
Makes sense! I guess the reason that it works in the x86_64 version is that GCC links the libstdc++ statically, potentially?